iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/VoltageLevelBusBreakerViewSwitchAdderAdapter.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.iidm.network.Switch;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class VoltageLevelBusBreakerViewSwitchAdderAdapter extends AbstractIdentifiableAdderAdapter<VoltageLevel.BusBreakerView.SwitchAdder> implements VoltageLevel.BusBreakerView.SwitchAdder {
```

Comment:
Maybe it's better to fit the IIDM API: SwitchAdder are nested classes of the relevant views

Reply:
I will make theses fields as nested classes